src/config/config.go

```diff
@@ -38,6 +38,8 @@ type ProxyConfig struct {
 	LongQueryTime    int    `json:"long-query-time"`
 	StreamBufferSize int    `json:"stream-buffer-size"`
 	IdleTxnTimeout   uint32 `json:"kill-idle-transaction"` //is consistent with the official 8.0 kill_idle_transaction
+
+	AutocommitFalseIsTxn bool `json:"autocommit-false-is-txn"`
```

Comment:
what's the AutocommitFalseIsTxn mean? 
Name confused me.

Reply:
Emmm, It is difficult to create this name. so reference the proxysql
[proxysql does not treat autocommit=0 as a transaction](https://github.com/sysown/proxysql/issues/1256)